js/main.js

```diff
@@ -4,22 +4,41 @@ var $addOptionButton = document.querySelector('.add-option-button');
 var $nutritionChoice = document.querySelector('#nutrients-choice');
 var $minMaxContainer = document.querySelector('.min-max-container');
 var $detailedRecipeContainer = document.querySelector('#detailed-recipe-container');
+var $favoriteIcon = document.querySelector('#favorite-icon');
+var $favoriteList = document.querySelector('#favorite-list');
+var $goToMainPageFromDetailed = document.querySelector('#go-to-main-page-detailed');
+var $goToMainPageFromFavorites = document.querySelector('#go-to-main-page-favorites');
 
 var sugarCount = 0;
 var proteinCount = 0;
 var carbsCount = 0;
 
 var selectNutritionName = '';
 var recipeName = '';
-var calories = '';
-var sugar = '';
-var protein = '';
-var carbs = '';
+var calories = 0;
+var sugar = 0;
+var protein = 0;
+var carbs = 0;
 var recipeImage = '';
 var amountOfServings = 0;
 var recipeId = '';
+var cholesterol = 0;
+var calcium = 0;
+var iron = 0;
+var potassium = 0;
+var magnesium = 0;
+var sodium = 0;
+var vitaminB6 = 0;
+var vitaminD = 0;
+var vitaminE = 0;
+var zinc = 0;
+var url = '';
+var ingredients = [];
+var detailedRecipeObject = {};
+var dataIdAttribute = '';
```

Comment:
Is it completely necessary to have all of these variables here? Can you just declare them in your function that uses them only?

https://github.com/max-cendalski/ajax-project/pull/11/files#diff-431e5adb210d9982cdf7bacdacad1bab1cd70c57028cb41316d8bb79d2d8b046R462